client/frontend/src/PreLogin/AboutUs.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect, Component } from 'react';
+import { ScrollTo } from "react-scroll-to";
+import LoginBox from '../Login/LoginBox';
+import { Redirect } from 'react-router-dom';
+import HomepageBackground from '../Login/HomepageBackground';
+import { makeStyles } from '@material-ui/core/styles';
+import axios from 'axios';
+import img1 from '../images/img1.jpeg';
+import img2 from '../images/img2.jpg';
+import img3 from '../images/img3.jpg';
+import img4 from '../images/img4.jpg';
+
+
+const useStyles = makeStyles((theme) => ({
+    wrapper: {
+        minHeight: '100vh'
+    },
+    header: {
+        width: '100%',
+        height: '70px',
+        marginTop: '10px',
+        borderBottom: '2px solid lightgrey',
+        paddingBottom: '10px'
+    },
+    font: {
+        textAlignVertical: "center",// <-- the magic
+        fontWeight: 'bold',
+        fontSize: 30,
+        paddingTop: '20px',
+        paddingBottom: '10px',
+        letterSpacing: '2px'
+    },
+    insideFont: {
+        padding: '10px',
+        fontSize: '27px',
+        marginLeft: '20px',
+        marginRight: '20px',
+        fontFamily: 'Vernon Adams',
+        fontWeight: 'bold',
+        paddingBottom: '10px',
+        letterSpacing: '1px' 
+    },
+    image: {
+        float: 'left',
+        marginLeft: '50px',
+        marginTop: '50px',
+    },
+    img: {
+        width: '300px',
+        height: '300px',
+    },
+    txtunder: {
+        teextAlign: 'center',
+        gridRow: '1',
+        fontFamily: 'Times New Roman'
+    }
+}));
+
+export default function AboutUs(props) {
+    const styles = useStyles();
+    return (
+         <div className={styles.wrapper}>
+                <div className={styles.header}>
+                    <p className={styles.insideFont}> 
+                        ChefMate is a web application that will allow users to search for recipes across popular <br></br>
+                        cooking websites and display results ranked according to their preferences. 
+                    </p>
+                 </div>
+                <div className={styles.image}>
+                <img src={img4} className={styles.img} />
+                <p className={styles.txtunder}> 
+                We have be ranked and recommended <br>
+                </br> recipes to our users according to sites that <br>
+                </br> they frequently visit through our engine.<br>
+                </br>Traditional search engines have seeds <br>
```

Comment:
Can you make the font for these bigger? Its too small right now. 
Also you can set a width on the p tag so you don't have to do the breaks manually.